docs/topolvm-node.md

```diff
@@ -26,9 +26,44 @@ If its response is succeeded, `topolvm-node` set `logicalvolume.status.volumeID`
 
 ### Finalize LogicalVolume
 
-When a `LogicalVolume` resource is being deleted, `topolvm-node` sends 
+When a `LogicalVolume` resource is being deleted, `topolvm-node` sends
 a `RemoveLV` request to `lvmd`.
 
+Inline ephemeral volume provisioning
+------------------------------------
+
+`topolvm-node` manages [inline ephemeral volumes](https://kubernetes-csi.github.io/docs/ephemeral-local-volumes.html) through special handling in the
+processing of `NodePublishVolumeRequest` and `NodeUnpublishVolume`. For volumes
+backed by PVCs, `topolvm-node` manages creation and deletion of LVMs via
+watches on `LogicalVolume` as explained above. However, for inline ephmeral
+volumes, the only CSI calls which will occur for the volume are
+`NodePublishVolumeRequest` and `NodeUnPublishVolumeRequest`. `topolvm-node` must
+create and delete the volumes in response to these calls.
+
+### Create an inline ephemeral volume
+
+Since `NodePublishVolumeRequest` and `NodeUnPublishVolumeRequest` are made for
+all volumes, `topolvm-node` must determine if the calls are for ephemeral
+inline volumes. It does so when processing a `NodePublishVolumeRequest` request
```

Comment:
"request" at the end of line is not necessary.

Reply:
done